sys/sys/jail.h

```diff
@@ -413,17 +413,17 @@ struct vfsconf;
  */
 #define jailed(cred)	(cred->cr_prison != &prison0)
 
-int jailed_without_vnet(struct ucred *);
+boolt jailed_without_vnet(struct ucred *);
```

Comment:
There seems to be a typo here. Should be just `bool`, not `boolt`.

Reply:
thanks for catching that

fixed!

Reply:
You are welcome, now we just need the changes to make it compile.

Reply:
looking okay now.
And would look a lot better with #668 merged.